From 0707b750c5d922ce5b835d00ba247101723373b7 Mon Sep 17 00:00:00 2001
From: Igor Ulyanov <disinbox@gmail.com>
Date: Sat, 9 Nov 2019 13:35:53 +0300
Subject: [PATCH] install headers

---
 library/openxlsx/CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/library/openxlsx/CMakeLists.txt b/library/openxlsx/CMakeLists.txt
index be94a6b..4577658 100644
--- a/library/openxlsx/CMakeLists.txt
+++ b/library/openxlsx/CMakeLists.txt
@@ -210,7 +210,10 @@ file(REMOVE_RECURSE ${OPENXLSX_INSTALLDIR}/include/)
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/interfaces/c++/headers/ DESTINATION ${OPENXLSX_INSTALLDIR}/include/OpenXLSX)
 file(COPY ${CMAKE_CURRENT_BINARY_DIR}/openxlsx_export.h DESTINATION ${OPENXLSX_INSTALLDIR}/include/OpenXLSX)
 
-
+set(OPENXLSX_HEADERS
+  ${OPENXLSX_CXX_INTERFACE_HEADERS}
+  ${CMAKE_CURRENT_BINARY_DIR}/openxlsx_export.h
+  )
 
 #=======================================================================================================================
 # Install OpenXLSX Library
-- 
2.23.0

